lib/improver/tests/utilities/test_temporal.py

```diff
@@ -452,6 +480,43 @@ def test_invalid_date_format(self):
                                forecast_time=6)
 
 
+class Test_extract_nearest_time_point(IrisTest):
+
+    """Test the extract_nearest_time_point function."""
+
+    def setUp(self):
+        """Set up a cube for the tests."""
+        cube = set_up_cube(num_time_points=2)
+        self.cube = add_forecast_reference_time_and_forecast_period(
```

Comment:
It would be nice to put a comment in saying what datetimes the numbers in the time coordinate correspond to

Reply:
Added a comment.